cosmoz-tabbable-behavior.html

```diff
@@ -210,7 +210,56 @@
 
 		_normalizeValue: function (value) {
 			return this.attrForSelected ? value : isNaN(value) || value === null ? value : Number(value);
+		},
+
+		/**
+		* True if the current element is visible.
+		*/
+		get _isVisible() {
+			return Boolean(this.offsetWidth || this.offsetHeight);
+		},
+
+		_onDescendantIronResize(event) {
+			if (this._notifyingDescendant || !this._isVisible) {
+				event.stopPropagation();
+				return;
+			}
+			const ev = Polymer.dom(event),
+				item =  ev.localTarget,
+				index = this.items.indexOf(item);
+
+			if (index < 0 || !item.isSelected) {
+				event.stopPropagation();
+				return;
+			}
+
+			if (!Polymer.Settings.useShadow) {
+				this._fireResize();
+			}
+		},
+
+		resizerShouldNotify(resizable) {
+			return resizable.is === 'paper-tabs' || resizable.isSelected;
+		},
+
+		_notifyTabResize() {
+			if (!this.isAttached) {
+				return;
+			}
+
+			this._interestedResizables.forEach(resizable => {
+				if (this.resizerShouldNotify(resizable)) {
+					this._notifyDescendant(resizable);
+				}
+			});
+
+			if (!this._isVisible) {
```

Comment:
Wouldn't it be also possible to not notify descendants when `this._isVisible` is false ?